lib/blueprinter/base.rb

```diff
@@ -68,13 +76,29 @@ def self.identifier(method, name: method, extractor: AutoExtractor)
     #     # other code
     #   end
     #
+    # @example Passing an if proc and unless method..
+    #   class UserBlueprint < Blueprinter::Base
+    #     def skip_first_name?(user, options)
+    #       user.first_name == options[:first_name]
+    #     end
+    #
+    #     field :first_name, unless: :skip_first_name?
+    #     field :last_name, if: ->(user, options) { user.first_name != options[:first_name] }
+    #     # other code
+    #   end
+    #
     # @return [Field] A Field object
     def self.field(method, options = {}, &block)
       options = if block_given?
         {name: method, extractor: BlockExtractor, block: {method => block}}
       else
         {name: method, extractor: AutoExtractor}
       end.merge(options)
+      %i[if unless].each do |condition|
+        if options[condition].is_a?(Symbol)
+          options[condition] = self.method(options.fetch(condition))
+        end
+      end
```

Comment:
There might be a way to resolve the conditional method at `render` execution time rather than at `field` declaration time, allowing conditional methods to be defined anywhere in the blueprint. Here's the idea, let me know what you think =)

What if you move this chunk of code into the `skip?` method you made in `Field`?